packages/storage/amplify_storage_s3_dart/lib/src/platform_impl/download_file/download_file_html.dart

```diff
@@ -40,14 +40,13 @@ S3DownloadFileOperation downloadFile({
 }
 
 Future<S3DownloadFileResult> _downloadFromUrl({
-  required String key,
+  required StoragePath path,
   required AWSFile localFile,
   required StorageDownloadFileOptions options,
   required S3PluginConfig s3pluginConfig,
   required StorageS3Service storageS3Service,
 }) async {
-  final s3PluginOptions = options.pluginOptions as S3DownloadFilePluginOptions;
-  final targetIdentityId = s3PluginOptions.targetIdentityId;
+  final s3PluginOptions = options.pluginOptions as S3DownloadFilePluginOptions?;
```

Comment:
@khatruong2009 do you recall if there was a reason that this was changed to nullable?